HVSample/src/main/java/com/microsoft/hsg/android/hvsample/FileUploadActivity.java

```diff
@@ -100,46 +100,46 @@ protected void onCreate(Bundle savedInstanceState) {
 		hvClient = new HealthVaultClient();
 		
 		progressContainer = findViewById(R.id.camera_progressContainer);
-        progressContainer.setVisibility(View.INVISIBLE);
-        
-        surfaceView = (SurfaceView)findViewById(R.id.camera_surfaceView);
-        SurfaceHolder holder = surfaceView.getHolder();
-        
-        holder.addCallback(new SurfaceHolder.Callback() {
+		progressContainer.setVisibility(View.INVISIBLE);
+
+		surfaceView = (SurfaceView)findViewById(R.id.camera_surfaceView);
+		SurfaceHolder holder = surfaceView.getHolder();
 
-            public void surfaceCreated(SurfaceHolder holder) {
-                try {
-                    if (camera != null) {
-                        camera.setPreviewDisplay(holder);
-                    }
-                } catch (IOException exception) {
-                    exception.printStackTrace();
-                }
-            }
+		holder.addCallback(new SurfaceHolder.Callback() {
 
-            public void surfaceDestroyed(SurfaceHolder holder) {
-                if (camera != null) {
-                    camera.stopPreview();
-                }
-            }
+			public void surfaceCreated(SurfaceHolder holder) {
```

Comment:
These methods are part of adding the callback "SurfaceHolder.Callback()" definition, and I think they are indentedcorrectly

---
In reply to: [117541763](https://github.com/Microsoft/healthvault-java-sdk/pull/5#discussion_r117541763) [](ancestors = 117541763)